serde_derive/src/ser.rs

```diff
@@ -275,6 +275,16 @@ fn serialize_variant(type_ident: &syn::Ident,
                                                     item_attrs,
                                                     tag)
             }
+            attr::EnumTag::InternalContent(ref tag, ref content) => {
+                serialize_internally_content_tagged_variant(type_ident.as_ref(),
```

Comment:
"Internally content tagged" doesn't makes sense to me. The internal vs external distinction has to do with the relationship between the tag and the content.

```rust
// externally tagged - tag is outside the content
{
  TAG: CONTENT
}

// internally tagged - tag is inside the content
{
  "type": TAG,
  CONTENT
}
```

By extension I would call this style adjacently tagged - the tag is adjacent to the content.

```rust
{
  "t": TAG,
  "c": CONTENT
}
```